massa-models/src/config/constants.rs

```diff
@@ -398,6 +398,8 @@ pub const MAX_DENUNCIATION_CHANGES_LENGTH: u64 = 1_000;
 
 /// Maximum number of events that can be returned by a query
 pub const MAX_EVENTS_PER_QUERY: usize = 10000;
+/// Delay between writes in event cache writer thread
+pub const EVENT_CACHE_TICK_DELAY: u64 = 100;
```

Comment:
Should these two constants be in config.toml instead (one node may want to increase / decrease these values based on their config / their needs)?

Reply:
For `EVENT_CACHE_TICK_DELAY` I agree (it should have almost no impact).
For `MAX_EVENTS_PER_QUERY` maybe it's more useful? (Depending on your performance tests).

Reply:
Fixed